kubel.el

```diff
@@ -256,14 +261,20 @@ CMD is the kubectl command as a list."
     (kubel--append-to-process-buffer
      (format "[%s]\ncommand: %s" process-name str-cmd))))
 
+(defun kubel--env-values ()
+  "Utility function to propgate kubernetes related environment variables."
+  (cl-map 'list (lambda(env) `(:name ,env :value ,(getenv env))) kubel-env-variables))
```

Comment:
silly question but why `cl-map` instead of `mapcar` or another non `cl-` function?

Reply:
No specific reason, I usually reach out to the `map` function. And from the Emacs reference, I learnt that `map` is deprecated and they are recommending to use `cl-map` instead.

`mapcar` seems to be a much better choice here for this case. Thanks!